gitcoinco/grants-stack
# Create Allo v2 Round

fixes https://github.com/gitcoinco/grants-stack/issues/2870

## How to test

1. Use the manager-staging preview to test allo v2 (NOT manager)
2. Ensure you are on Allo v2 by checking the footer
3. Ensure you have a program (profile) created
4. Trying to create a program on V2 will show up a message saying to go to Builder
5. If you don't have a program created, go to Builder staging to create it (https://grants-stack-builder-staging.vercel.app/#/projects), then come back to Manager (in Allo v2 a project is a program)
6. You should see your program now
7. Create a QF or DG round using Allo V2, only testnets should work (I am testing on Mumbai)
8. You should see the round you created in the list of rounds for the program
9. In the round list, verify that the round appears with the correct title, dates
10. QF rounds should have a QF badge. DG rounds are still not indexed properly and will not have a badge, waiting on this PR: https://github.com/gitcoinco/grants-stack-indexer/pull/451

Notes:

- Clicking the round itself to see the round page will not work yet, this is being done here: https://github.com/gitcoinco/grants-stack/pull/2935



## Checklist

This PR:
